### DISCUSSION TITLE
Does HorizontalRunnerAutoScaler work in 2024?

### DISCUSSION THREAD

**Discussion by marverix** [actions/actions-runner-controller#3524] [Questions]
Hello,
I'm new to this community and I didn't know how to properly ask the question. I don't know the whole history behind this project, but as far as I can see, it looks for me like it was previously some 3rd party project (in domain summerwind.dev), that had already version 0.23.x, and something happened couple (?) mounts ago and now it's official GitHub project, and from unknown for me reason, it started to the version from 0. So now you can find on the Internet `actions-runner-controller` documentation pointing to version 0.23, but on the official repo, there is version 0.9.1. Indeed fascinating! But, what's more confusing for me, is that now some CRDs are in domain actions.summerwind.dev, and some on new domain actions.github.com. Here and there one can find notes that, yeah documentation to the legacy version is here, but it seems for me that legacy documentation was more reach and detailed. And now, to the point: For example HorizontalRunnerAutoScaler, which is the most interesting for me, is only documented in the legacy documentation. On "new" documentation not a word about it. But in the source code, one can still find the CRDs for HorizontalRunnerAutoScaler. 
So... is it working in 2024? In the current new (0.9.1) version? Or has it been dropped and the source code is only for grandchildren? If it's working, is it somewhere documented?
Thanks in advance for the answer :)

**Comment by marverix** [comment]
Up

**Reply by crohr** [reply]
@marverix I was so frustrated with ARC and the Philipps terraform provider that I went ahead and created another project for self-hosted runners. I would recommend you have a look at [RunsOn](https://runs-on.com), which handles auto-scaling and provides compatible base images for both x64 and arm64 architectures (all on-premise).

**Reply by marverix** [reply]
Thanks, this is for sure an alternative that me, and my team will look at

**Comment by marverix** [comment]
Up
